docs/user/reference/openapi.yaml

```diff
@@ -240,6 +275,31 @@ paths:
                 $ref: '#/components/schemas/HTTPValidationError'
           description: Validation Error
       summary: Submit Task
+  /tasks/{task_id}:
+    get:
+      description: Retrieve a task
+      operationId: get_task_tasks__task_id__get
+      parameters:
+      - in: path
+        name: task_id
+        required: true
+        schema:
+          title: Task Id
+          type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TrackableTask'
+          description: Successful Response
+        '422':
```

Comment:
After talking with Rose, we actually return 422 in many places, so it's OK to return this.
But we do also want the 404 to be called out here, especially as it's raised from the HTTPException in the method it's strange it hasn't caught it. Possibly in the decorater.responses={status.HTTP_404_NOT_FOUND: ["item not found"]}?